ui/core/components/inputs/consumables.ts

```diff
@@ -806,7 +807,9 @@ export const makeMp5ConsumeInput = makeConsumeInputFactory({ consumesFieldName:
 export const Windfury: ConsumableInputConfig<WeaponImbue> = {
 	actionId: () => ActionId.fromSpellId(10614),
 	value: WeaponImbue.Windfury,
-	showWhen: player => player.getFaction() === Faction.Horde,
+	showWhen: player => {
+		return (player.getFaction() === Faction.Horde) && !player.isSpec(Spec.SpecFeralDruid)
```

Comment:
So for correctness, we'd rather make the ability itself not work while shapeshifted, rather than just disable having access to it for Feral. Same for all these imbues. Since technically someone might want to do some weird melee-weave type rotation and it should "just work".

Or, more practically, I can show people the negative effects of queueing a shapeshift on the GCD which leads to minor caster form uptime/melees.

Though actually maybe it's correct to just not even show them as an option since they will not have the effect most users expect.

Reply:
Discussed in the wowsims discord and decided to do both - they no longer work while shapeshifted and they also don't show up in the UI.